src/features/whereAreWe/infoBox/infoBox.tsx

```diff
@@ -0,0 +1,31 @@
+import clsx from "clsx";
+import React from "react";
+import styles from "./infoBox.module.scss";
+
+interface InfoBoxProps {
+  text: string;
+  variant: "green" | "blue" | "lime" | "skyBlue";
+  separator?: string;
+}
+
+export function InfoBox({ text, variant, separator }: InfoBoxProps) {
```

Comment:
can you add unit tests for it? I don't get how it works exactly

Reply:
done